src/fw-update/fw-update-factory.cpp

```diff
@@ -48,8 +115,16 @@ namespace librealsense
                 auto usb = platform::usb_enumerator::create_usb_device(info);
                 if (!usb)
                     continue;
-                if (ds::RS_RECOVERY_PID == info.pid || ds::RS_USB2_RECOVERY_PID == info.pid)
+                if (ds::RS_RECOVERY_PID == info.pid)
                     return std::make_shared<ds_update_device>(ctx, register_device_notifications, usb);
+                if (ds::RS_USB2_RECOVERY_PID == info.pid || L500_USB2_RECOVERY_PID_OLD == info.pid)
```

Comment:
Aren't they the same value?

Reply:
They are.
I wanted to emphasize that it could be an L515 or a DS.
Just to clarify that this value has 2 meanings..